src/meta/meta_ckpt.c

```diff
@@ -835,11 +836,14 @@ __wt_meta_ckptlist_get(
             WT_ERR(ret);
         }
     } else {
-        WT_ERR(__wt_metadata_search(session, fname, &config));
+        WT_ERR(__wt_btree_shared_base_name(session, &fname, NULL, &name_buf));
+        WT_ERR(__wt_metadata_search(session, name_buf == NULL ? fname : name_buf->data, &config));
```

Comment:
`name_buf == NULL ? fname : name_buf->data` can be replaced by `fname`, because `fname` is updated by `__wt_btree_shared_base_name`.